osu.Game/Tests/Visual/Multiplayer/TestMultiplayerClient.cs

```diff
@@ -114,12 +116,24 @@ public void ChangeRoomState(MultiplayerRoomState newState)
         public void ChangeUserState(int userId, MultiplayerUserState newState)
```

Comment:
I guess it's safe to ignore most of the implementation detail in this class as being duplicate/best-effort implementation of a local server?

Reply:
Best-effort, yes.